### PR TITLE
I added aria-label to the buttons

### LINKED ISSUE #969: [Accessibility] Slides Page: Buttons do not have accessible name
**Description**
The download and delete buttons do not have accessible names. When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers.

**Screenshots**
image

**Proposed Fix**
Add an `aria-label` attribute to the button elements

### PR DESCRIPTION
**Pull request by Joice-crypto**
This PR fixes  #969

## Motivation
I added an arial-label to the delete and download buttons to be recognized in the screen readers.

## screenshot

Captura de tela de 2024-07-01 22-35-39

### CODE REVIEW

File: apps/table.js
Comment (birm) [inline apps/table.js:339]:
```suggestion
                    <button aria-label="Delete" type='button' class='btn btn-danger btn-sm DelButton' id='deleteBtn' data-id='${sanitize(rs[0])}' data-name='${sanitize(rs[1])}' onclick='deleteSld(this)' data-filename='${sanitize(filename)}' data-toggle='modal'>
```

**Comment by nwanduka** [comment]
Thanks a lot @Joice-crypto for working on this. The changes look good to me. We'll wait for Birm to give a final review, approve, and merge.

**Review by birm — APPROVED** [review]
Thank you!

**Pull request merged**
